Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,65 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Globalization;
 using Console = System.Console;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
+        public static string firstName;
+        public static string surname;
+        public static int age;
+        public static float weight;
+        public static float height;
+        public static float bmi;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'bmi' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845553)